diffhtwo/experimental/precompute_emission_line_luminosity_kern.py

```diff
@@ -0,0 +1,199 @@
+import jax.numpy as jnp
+import numpy as np
+import numpy.ma as ma
+from jax import grad
+
+# copied from astropy.constants.c.value
+C = 299792458.0
```

Comment:
Try to remember to add in the units with an in-line comment whenever hard-coding a global constant like this.